app/models/concerns/translated_content.rb

```diff
@@ -3,7 +3,7 @@ module TranslatedContent
   include RoleControl::ParentalControlled
 
   included do
-    has_paper_trail skip: [:langauge]
+    has_paper_trail skip: [:language]
```

Comment:
agree it's weird not to change it while you can but i think it's out of scope for this PR. Do the tests fail if you revert it?. 5/10 on this one.